src/components/Summary/AlterationSummary.vue

```diff
@@ -88,10 +89,6 @@
       </div>
     </template>
 
-    <template v-if="hasFolioNumberChanged">
-      <!-- TODO: implement this (#5024) -->
```

Comment:
Changed Folio Number does not display on alteration summary page.
(Transactional FN -- future PR -- will show on this page in a new section.)